docs/UserGuide.md

```diff
@@ -758,6 +758,7 @@ Note that the last example is **not equivalent** to `n/do && t/friend || t/colle
 7. **When executing the `find` command**, if you input a `FIND_EXPRESSION` that is not accepted by the command format, the error message shows fairly general error messages. We will be working on this in the future to **improve the specificity of error messages.**
 8. **When executing the `find` command**, it is impossible to search for keywords that include the double-quote character (`"`) under any circumstance. We will be working on this in the future to **support searching for the double-quote character**, which could appear in fields such as notes.
 9. **When executing the `find` command**, the behavior of the `bal` field is not intuitive, especially for users who do not read the User Guide in-depth. We will be working on this in the future to **improve the ergonomics of `bal` field**, by implementing `>` and `<` operators so users can search for balance amounts below or above the keywords.
+10. **When starting the app**, if data that is previously stored in `addressbook.json` is unable to be parsed by the application as a valid `AddressBook` (e.g. data is corrupt), the app may overwrite it on the next exit, causing data loss. We will rewrite our data storage code to be more robust when dealing with this issue by backing up data files even when they are corrupt, allowing users to retrieve them in the future. 
```

Comment:
could you remove this diff? since the section will be removed once Calvin's currently-approved PR is merged anyways (and this diff would cause merge confclit)